arm-network/2017-06-01/swagger/azureWAN.json

```diff
@@ -0,0 +1,2434 @@
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/Hub"
+                        },
+                        "description": "Parameters supplied to create or update Hub."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the Hub created or updated.",
+                        "schema": {
+                            "$ref": "#/definitions/Hub"
+                        }
+                    },
+                    "202": {
+                        "description": "Request received successfully. Hub is not yet provisioned; follow the Location header to poll for final outcome."
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            },
+            "patch": {
+                "operationId": "Hubs_Update",
+                "x-ms-examples": {
+                    "HubUpdate": { "$ref": "../examples/HubUpdate.json" }
+                },
+                "description": "Updates a Hub.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/HubParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/Hub"
+                        },
+                        "description": "Parameters supplied to update Hub."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the Hub updated.",
+                        "schema": {
+                            "$ref": "#/definitions/Hub"
+                        }
+                    },
+                    "202": {
+                        "description": "Request received successfully. Hub is not yet updated; follow the Location header to poll for final outcome."
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            },
+            "delete": {
```

Comment:
Does delete on the hub delete the scalableExpressRouteGateways and scalableVpnGateways under it?